package.json

```diff
@@ -22,7 +22,7 @@
     "test": "NODE_ENV=unit-test NODE_OPTIONS=--experimental-vm-modules jest --runInBand --detectOpenHandles --collectCoverage=true --testPathPattern='(server|frontend)/.*'",
     "security_audit": "better-npm-audit audit",
     "int-test": "cypress run --config video=false",
-    "int-test-ui": "cypress open",
+    "int-test-ui": "cypress open --config watchForFileChanges=false",
```

Comment:
I prefer to refresh my own Cypress tests in Cypress UI.

Can remove if contentious